Source/Modules/delphi.cxx

```diff
@@ -4329,25 +4353,25 @@ class DELPHI:public Language {
 
 				  Swig_restore(n);
 			  }
-			  else if (Cmp(bb, "void") == 0 ) { 
+			  else if (Cmp(bb, "void") == 0 ) {
 
 				  /* since this is a recurring issue, we are going to remember the
 				  typedef pointer, if already it is not a pointer or reference, as
 				  in
 
 				  typedef void NT;
-				  int func(NT *p); 
+				  int func(NT *p);
 
 				  see director_basic.i for example.
 				  */
 
 				  String *record = NewString("");
 
 
-				  Printf(record, 
-					  "   $RecordName = type pointer;\n"
+				  Printf(record,
+					  "   $RecordName = type pointer; 1111\n"
```

Comment:
It seems to be the forgotten debugging code. What is the purpose of this change?